language/tools/move-cli/src/main.rs

```diff
@@ -775,7 +778,33 @@ fn main() -> Result<()> {
                 *dry_run,
             )
         }
-        Command::Test { path, track_cov } => test::run_all(
+        Command::Test {
+            path,
+            track_cov: _,
+            create: true,
+        } => {
+            let dirs = ["src", "src/modules", "src/scripts"];
+            let files = ["args.txt"];
```

Comment:
Might be useful to put a comment or two at the top of args.txt that explains what to do; e.g.,

```
# This is a batch file. To write an expected value test that runs `move <command`> <args1>; move <command2> <args2>, write <command1> <args1>\n<command2> <args2>`. # is a comment
```

Reply:
Good idea! I've added it in.